package.json

```diff
@@ -62,11 +62,13 @@
     "@types/tar": "^6.1.4",
     "fastify": "^3.29.4",
     "fastify-formbody": "^5.2.0",
+    "google-auth-library": "^9.10.0",
     "js-yaml": "^4.1.0",
     "knex": "^2.4.2",
     "middie": "^5.3.0",
     "module-alias": "^2.2.2",
     "moment-timezone": "^0.5.39",
+    "node-fetch": "2",
```

Comment:
is it possible to use a later version here?

Reply:
I had issues with newer versions as our code isn't fully compatible with ESM-only modules and v2 is compatible with CommonJS.

Reply:
Bumped it to the latest version that is supported